### PR TITLE
Add cloud trial operations workflow and expectations

### PR DESCRIPTION
**Pull request by malomarrec**
This workflow is done, excep the qualification criteria(future PR), and will be announced on Monday.

### CODE REVIEW

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+      1. an initial password reset link to the instance
+      2. a link to the onboarding checklist
+      3. a link to schedule time with a customer engineer (CE)
+   3. set the lead status in salesforce to `Trial Instance Handed Off`
+
+### PQL qualification workflow
+
+The product growth team will be monitoring analytics for trial instances that have usage patterns that show signs that they could convert into a customer: Product Qualified Leads (PQL). For now, the criteria is best judgement and the workflow is manual, but this will be clearly defined and automated in the future.
+
+When a trial becomes a PQL, the lead status will be changed to `PQL` in salesforce, which will trigger an alert in #channel-TBD, to be picked up by an Account Executive.
```
Comment (malomarrec) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]: @ngage1  in what channel should we alert when a trial becomes a PQL?

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+- 5 days max after the 30-day trial ends, make sure a decision is made to [extend, convert, or terminate](../../../../departments/cloud/trial_mi.md/#cloud-trial-managed-instances) the trial
+- monitor metrics for product qualified leads (PQLs)
+- (those are just operational tasks, the product growth team is also working on its usual produc team stuff)
+
+#### Long term
+
+- Automate it all!
+- Also see our [broader strategy](../../../../strategy-goals/strategy/growth-team/index.md).
+
+## Customer Engineering (CE)
```
Comment (malomarrec) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md:86]: @amenne this is what we discussed yesterday. Does that work for you?

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+# Cloud trial operations
+
+From 2022-09-27, Sourcegraph Cloud will be the default way of deploying Sourcegraph. All the call to actions (CTAs) on Sourcegraph sites and posts will let users request a 30-day cloud trial. This page describes the workflow and expectations from each team to make sure those trial users get a great experience. It also describes what the role of each team will be in the longer term.
+
+## Context
+
+Cloud is the default way of deploying Souregraph, but there are some limits to delivering a cloud trial instance today.
+
+- There is a cap on the number of instances we can managed at any given time ([SLA](#TODO)). Therefore, when someone requests an instance, we have to qualify them to make sure that we have capacity and that a cloud instance is the best option to meet their needs.
+- Cloud instances take some time to provision ([SLA](#TODO)), that makes it so signing up for a cloud instance is not instance.
```
Comment (malomarrec) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]: This is pending this question: https://sourcegraph.slack.com/archives/C03JR7S7KRP/p1663789661106919

Comment (ebrodymoore) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]:
```suggestion
- Cloud instances take some time to provision ([SLA](#TODO)), that makes it so signing up for a cloud instance is not instant.
```

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    - set the lead status to `Approved trial`
+    - if the requesting user is from an existing customer or on the [strat 100](https://docs.google.com/spreadsheets/d/1JFHacGYDIBd4pMSrKC3QV25YFkK2yBfM0dMd9An2sGE/edit#gid=637855099) list, notify the owning Account Executive
+    - 🟢 your job as a SDR is done!
+    - this will trigger an alert in #cloud-trial-alerts channel and start the [Instance provisioning and hand-off workflow](instance-provisioning-and-hand-off-workflow)
+- If there **is** already an instance on the domain of the requesting user's email
+  - set the lead status to `disqualified`
+  - set the lead substatus to `Domain Already in Trial`
+  - 🟢 your job as a SDR is done!
+  - this will send an alert in #cloud-trial-alerts and automatically send an email to the user to let them know about the pre-existing instance.
+  - if SSO is setup on that instance, the user can log in. If not, they can't. Therefore, @malo or @Eric Brody-Moore will email the site-admin to let them know someone tried to join. (This is a quick fix until there is a "request an invite" system).
```
Comment (ebrodymoore) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]:
```suggestion
  - if SSO is setup on that instance, the user can log in. If not, they can't. Therefore, `@malo` or `@Eric Brody-Moore` will email the site-admin to let them know someone tried to join. (This is a quick fix until there is a "request an invite" system).
```

File: content/departments/engineering/teams/growth/cloud-trial-operations.md (hunk 2 above)
Comment (amenne) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md:86]: CE side looks good. For Support I’d say our [normal SLAS](https://handbook.sourcegraph.com/departments/ce-support/support/#sts=SLAs) for now and we’ll revisit depending on quantity. (Before we have to go build and track a separate set).

Not sure where this fits but we talked about when a customer opts-in to 30 mins with CE we would generate a msg in #ce and tag the @ce-team. Right?

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+### Instance provisioning and hand-off workflow
+
+@ebrodymoore and @malo will be monitoring the #cloud-trial-alerts for qualified requests. When a request is sent, they will:
+
+1. self-assign the request by replying "I'm taking it" to the alert
+1. raise a trial instance request with the cloud team (this step will be automated soon)
+1. when the instance is ready
+   1. set a license on the instance with 1,000 users, a 30-day limit, and the following tags: `plan:enterprise-0`,`acls`,`private-extension-registry`,`remote-extensions-allow-disallow`,`monitoring`,`true-up`,`batch-changes`,`code-insights`,`trial`
+   2. send the admin an onboarding email with
```
Comment (ebrodymoore) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md]: We should have a link to the email templates (or a placeholder)

**Review by amenne — APPROVED** [review]

File: content/departments/engineering/teams/growth/cloud-trial-operations.md (hunk 2 above)
Comment (malomarrec) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md:86]: Thanks.
@ngage1 can you build an automate that workflow? (with "chili something", if don't remember exactly)

File: content/departments/engineering/teams/growth/cloud-trial-operations.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+      1. an initial password reset link to the instance
+      2. a link to the onboarding checklist
+      3. a link to schedule time with a customer engineer (CE)
+   3. set the lead status in salesforce to `Trial Instance Handed Off`
+
+### PQL qualification workflow
+
```
Comment (ebrodymoore) [inline content/departments/engineering/teams/growth/cloud-trial-operations.md:54]:
```suggestion

`@Eric Brody-Moore` to  complete this section with more specifics

```

**Review by ebrodymoore — APPROVED** [review]
Seems really good, awesome work

File: content/departments/engineering/teams/growth/cloud-trial-operations.md (hunk 1 above)
Comment (ngage1) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md]: @malomarrec let's use #product-qualified-leads

File: content/departments/engineering/teams/growth/cloud-trial-operations.md (hunk 2 above)
Comment (ngage1) [reply content/departments/engineering/teams/growth/cloud-trial-operations.md:86]: @seth-sourcegraph can you please include the CP link to schedule these 30 minute syncs- Dan and Casi can then loop in the CE.

**Comment by malomarrec** [comment]
I am merging this, with the expectation that the qualification status will be updated in a separate PR before Tuesday.

**Comment by malomarrec** [comment]
I am merging this, with the expectation that the qualification status will be updated in a separate PR before Tuesday.

**Pull request merged**